Doc/library/dataclasses.rst

```diff
@@ -344,6 +344,14 @@ Module contents
    Other attributes may exist, but they are private and must not be
    inspected or relied on.
 
+.. class:: InitVar
+
+   ``InitVar[T]`` type hints describe variables that are
+   :ref:`init-only <dataclasses-init-only-variables>`. Fields annotated with :class:`!InitVar`
```

Comment:
The purpose of my rewording was to be sure that we don't exceed 80 chars per line, including indentation. Note that Sphinx supports splitting before `<` in `ref` so its' fine.